app/src/main/java/io/intrepid/contest/models/Category.java

```diff
@@ -0,0 +1,27 @@
+package io.intrepid.contest.models;
+
+public class Category {
+    private String name;
+    private String description;
+
+    public Category(String name, String description) {
+        this.name = name;
+        this.description = description;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getDescription() {
```

Comment:
Get and Set description are not used. I would normally recommend removing them but where you are probably going to use them almost immediately I think it will be ok. Its not a good habit to get into though.